app/controllers/finders_controller.rb

```diff
@@ -2,10 +2,10 @@ class FindersController < ApplicationController
   layout "finder_layout"
 
   before_action do
-    expires_in(5.minutes, public: true)
+    @content_item ||= ContentItem.from_content_store(finder_base_path)
+    expires_in(content_item.max_age, public: content_item.cache_public)
```

Comment:
On line 82 there's a `content_item` method you could use instead, so you could delete line 5.

```ruby
def content_item
    @content_item ||= ContentItem.from_content_store(finder_base_path)
  end
```

Reply:
👍